roles/installer/templates/cronjobs/metrics-gather.yaml.j2

```
@@ -0,0 +1,56 @@
+---
+apiVersion: v1
```

I never know if we should be explicit on the api version value since cronjobs are using batch

```suggestion
apiVersion: batch/v1
```